src/styles/stars.css

```diff
@@ -0,0 +1,19 @@
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  height: 16px;
+  width: 16px;
+  background-image: url(../images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars__star:not(:last-child) {
+  margin-right: 4px;
+}
+
+.stars__star:nth-child(-n+4) {
+  background-image: url(../images/star-active.svg);
+}
```

Comment:
![image](https://user-images.githubusercontent.com/98764641/217778129-fb22d2aa-2f31-4f5c-bd37-406a8876aba8.png)

Reply:
What needs to be changed here? I wrote the code as I understood the task. Direct in which direction to look for the answer.